sources/platform/console/two-factor-authentication.md

```diff
@@ -0,0 +1,97 @@
+
+## Setting up two-factor authentication
+
+To set up two-factor authentication, go to your [account settings](https://console.apify.com/account/settings) and scroll down to the **Two-factor authentication** section. Currently, there is only one option, which is the **Authenticator app**. If you have two-factor authentication already enabled, there will be a label **enabled** next to it.
+
+![Apify console two-factor authentication section](./images/console-account-two-factor-disabled.png)
+
+If it's not enabled click on the **Enable** button you should see the two-factor setup view.
+
+### Authenticator app setup
+
+![Apify console setup two-factor authentication - app](./images/console-two-factor-app-setup.png)
+
+In this view, you can use your favorite authenticator app to scan the QR code. We recommend using Google Authenticator ([Google play store](https://play.google.com/store/apps/details?id=com.google.android.apps.authenticator2&hl=en_US)/[Apple app store](https://apps.apple.com/us/app/google-authenticator/id388497605)) or [Authy](https://authy.com/) but any other authenticator app should work as well.
+
+You can also set up your app/browser extension manually without the QR code. To do that, click on the **Setup key** link below the QR code this view with the key will pop up:
+
+![Apify console setup two-factor authentication - key](./images/console-setup-two-factor-auth-key.png)
+
+Here you can copy the two-factor `secret` key, which you can use in your authenticator app to set up the account. You can also use this key to set up your authenticator app on multiple devices.
+
+After you scan the QR code or set up your app manually, the app will generate a code which you need to enter into the **Verify the code from the app** field. After you enter the code, click on the **Continue** button to get to the next step of the setup process.
+
+### Recovery settings
+
+![Apify console setup two-factor authentication - recovery codes](./images/console-two-factor-recovery-setup.png)
+
+In this step, you will see 16 recovery codes. If you ever lose access to your authenticated app, you will be able to use these codes to access the Apify Console. We recommend saving these codes in a safe place, ideally, you should either store them in a secure password manager or print them out and keep them separate from your device.
+
+Under the recovery codes, you will find two fields for your recovery information. These two fields are what the support team will ask you to provide in case you lose access to your authenticator app and also to your recovery codes. We will never use the phone number for anything else than to verify your identity and help you regain access to your account and only as a last resort. Ideally, the personal information provided by you will be enough to verify your identity. Always provide both what kind of personal information you provide and the actual information.
```

Comment:
```suggestion
Under the recovery codes, you will find two fields for your recovery information. These two fields are what the support team will ask you to provide in case you lose access to your authenticator app and also to your recovery codes. We will never use the phone number for anything other than to verify your identity and help you regain access to your account, only as a last resort. Ideally, the personal information you provide will be enough to verify your identity. Always provide both the kind of personal information you provide and the actual information.
```